--- ../src/eplus/handlers/CapeTickHandler.java
+++ ../src_work/eplus/handlers/CapeTickHandler.java
@@ -9,7 +9,10 @@
 
 import net.minecraft.client.Minecraft;
 import net.minecraft.client.renderer.ImageBufferDownload;
+import net.minecraft.client.renderer.ThreadDownloadImageData;
+import net.minecraft.client.renderer.texture.TextureObject;
 import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.ResourceLocation;
 import cpw.mods.fml.client.FMLClientHandler;
 import cpw.mods.fml.common.ITickHandler;
 import cpw.mods.fml.common.TickType;
@@ -54,6 +57,12 @@
         {
             EnchantingPlus.log.warning("Could not load capes from remote authority.");
         }
+        
+        for (String modder : modders)
+        {
+             ThreadDownloadImageData object = new ThreadDownloadImageData(capeURL, null, null);
+             mc.renderEngine.loadTexture(new ResourceLocation("cloaks/" + modder), (TextureObject) object);
+        }
     }
 
     @Override
@@ -65,33 +74,6 @@
     @Override
     public void tickEnd(EnumSet<TickType> type, Object... tickData)
     {
-
-        if (mc.theWorld != null && mc.theWorld.playerEntities.size() > 0)
-        {
-            @SuppressWarnings("unchecked")
-            List<EntityPlayer> players = mc.theWorld.playerEntities;
-
-            for (EntityPlayer player : players)
-            {
-                if (player != null)
-                {
-                    for (String modder : modders)
-                    {
-                        if (modder.equalsIgnoreCase(player.username) && player.cloakUrl.startsWith("http://skins.minecraft.net/MinecraftCloaks/"))
-                        {
-
-                            String oldCloak = player.cloakUrl;
-                            player.cloakUrl = capeURL;
-                            if (!player.cloakUrl.equals(oldCloak))
-                            {
-                                mc.renderEngine.obtainImageData(player.cloakUrl, new ImageBufferDownload());
-                            }
-                        }
-                    }
-                }
-            }
-
-        }
 
     }
 
