--- ../src/eplus/handlers/VersionTickHandler.java
+++ ../src_work/eplus/handlers/VersionTickHandler.java
@@ -3,6 +3,7 @@
 import java.util.EnumSet;
 
 import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.ChatMessageComponent;
 import cpw.mods.fml.common.IScheduledTickHandler;
 import cpw.mods.fml.common.TickType;
 import eplus.lib.References;
@@ -60,10 +61,15 @@
         {
             if (Version.hasUpdated())
             {
-                player.sendChatToPlayer(String.format("[%s]: %s: %s", References.MODID, "Version update is available", Version.getRecommendedVersion()));
-                player.sendChatToPlayer(String.format("[%s]: to view a changelog use: %s", References.MODID, "/eplus changelog"));
+                sendChatToPlayer(player, String.format("[%s]: %s: %s", References.MODID, "Version update is available", Version.getRecommendedVersion()));
+                sendChatToPlayer(player, String.format("[%s]: to view a changelog use: %s", References.MODID, "/eplus changelog"));
             }
         }
         messageSent = true;
     }
+
+    private void sendChatToPlayer(EntityPlayer player, String format)
+    {
+        player.sendChatToPlayer(ChatMessageComponent.createFromText(format));
+    }
 }
