--- ../src/eplus/commands/EplusCommands.java
+++ ../src_work6/eplus/commands/EplusCommands.java
@@ -7,6 +7,7 @@
 import net.minecraft.command.CommandBase;
 import net.minecraft.command.ICommandSender;
 import net.minecraft.command.WrongUsageException;
+import net.minecraft.util.ChatMessageComponent;
 import cpw.mods.fml.common.network.PacketDispatcher;
 import eplus.EnchantingPlus;
 import eplus.handlers.ConfigurationHandler;
@@ -77,12 +78,17 @@
 
     private void processChangelog(ICommandSender icommandsender, String commandName, String[] args)
     {
-        icommandsender.sendChatToPlayer(String.format("\u00A7e[%s] Changelog for %s", References.MODID, Version.getRecommendedVersion()));
+        sendChatToPlayer(icommandsender, String.format("\u00A7e[%s] Changelog for %s", References.MODID, Version.getRecommendedVersion()));
         for (final String line : Version.getChangelog())
         {
-            icommandsender.sendChatToPlayer(line);
+            sendChatToPlayer(icommandsender, line);
         }
 
+    }
+
+    private void sendChatToPlayer(ICommandSender icommandsender, String line)
+    {
+        icommandsender.sendChatToPlayer(ChatMessageComponent.createFromText(line));
     }
 
     @Override
@@ -129,7 +135,7 @@
                 config.put(commandName, args[0]);
 
                 PacketDispatcher.sendPacketToServer(new ConfigPacket(config).makePacket());
-                icommandsender.sendChatToPlayer(String.format("%s: Config '%s' changed to %s.", References.MODID.toUpperCase(), commandName, args[0]));
+                sendChatToPlayer(icommandsender, String.format("%s: Config '%s' changed to %s.", References.MODID.toUpperCase(), commandName, args[0]));
                 return;
             }
         }
