--- ../eplus/EnchantingPlus.java
+++ ../work/eplus/EnchantingPlus.java
@@ -44,7 +44,7 @@
  * @license Lesser GNU Public License v3 (http://www.gnu.org/licenses/lgpl.html)
  */
 
-@Mod(name = References.MODNAME, modid = References.MODID, dependencies = "required-after:Forge@[7.8.0.684, 7.8.1.738]", acceptedMinecraftVersions = "[1.5.2, 1.6.1)")
+@Mod(name = References.MODNAME, modid = References.MODID, dependencies = "required-after:Forge@[7.10.0.789,)", acceptedMinecraftVersions = "[1.6.2, )")
 @NetworkMod(channels =
 { BasePacket.CHANNEL }, packetHandler = PacketHandler.class, connectionHandler = ConnectionHandler.class, clientSideRequired = true)
 public class EnchantingPlus
