--- ../src/eplus/gui/GuiModTable.java
+++ ../src_work6/eplus/gui/GuiModTable.java
@@ -12,6 +12,7 @@
 import net.minecraft.enchantment.Enchantment;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.entity.player.InventoryPlayer;
+import net.minecraft.util.ResourceLocation;
 import net.minecraft.world.World;
 
 import org.lwjgl.input.Mouse;
@@ -278,7 +279,7 @@
 
     private boolean dirty = false;
 
-    private final String texture = "/assets/eplus/textures/gui/enchant.png";
+    private final ResourceLocation texture = new ResourceLocation("eplus", "textures/gui/enchant.png");
 
     public String error = "";
 
