--- ../eplus/gui/GuiModTable.java
+++ ../work/eplus/gui/GuiModTable.java
@@ -12,6 +12,7 @@
 import net.minecraft.enchantment.Enchantment;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.entity.player.InventoryPlayer;
+import net.minecraft.util.ResourceLocation;
 import net.minecraft.world.World;
 
 import org.lwjgl.input.Mouse;
@@ -89,7 +90,7 @@
                 GL11.glDisable(GL11.GL_LIGHTING);
                 GL11.glDisable(GL11.GL_DEPTH_TEST);
                 GL11.glColor4f(1.0f, 1.0f, 1.0f, 1.0f);
-                mc.renderEngine.bindTexture(texture);
+                mc.renderEngine.func_110577_a(texture);
                 drawTexturedModalRect(xPosition, yPosition, 8 + textureIndex * 16, 182, width, height);
             }
         }
@@ -273,9 +274,11 @@
 
     private boolean dirty = false;
 
-    private final String texture = "/assets/eplus/textures/gui/enchant.png";
+    private final ResourceLocation texture = new ResourceLocation("eplus", "gui/enchant.png");
 
     private String error = "";
+    
+    private static int guiOffset = 26;
 
     public GuiModTable(InventoryPlayer inventory, World world, int x, int y, int z, TileEnchantTable tileEntity)
     {
@@ -288,7 +291,7 @@
         yPos = y;
         zPos = z;
 
-        xSize = 209;
+        xSize = 235;
         ySize = 182;
     }
 
@@ -369,7 +372,7 @@
     protected void drawGuiContainerBackgroundLayer(float f, int i, int j)
     {
 
-        mc.renderEngine.bindTexture(texture);
+        mc.renderEngine.func_110577_a(texture);
         drawTexturedModalRect(guiLeft, guiTop, 0, 0, xSize, ySize);
 
         for (final GuiItem item : enchantmentArray)
@@ -395,7 +398,7 @@
         final int adjustedMouseX = par1 - guiLeft;
         final int adjustedMouseY = par2 - guiTop;
 
-        mc.renderEngine.bindTexture(texture);
+        mc.renderEngine.func_110577_a(texture);
         GL11.glEnable(GL11.GL_LIGHTING);
         GL11.glEnable(GL11.GL_DEPTH_TEST);
         GL11.glColor4f(1.0f, 1.0f, 1.0f, 1.0f);
@@ -418,7 +421,7 @@
 
         sliderY = sliding ? tempY : 57 * (sliderIndex / enchantingPages);
 
-        drawTexturedModalRect(guiLeft + 180, guiTop + 16 + (int) sliderY, 0, 182, 12, 15);
+        drawTexturedModalRect(guiLeft + guiOffset + 180, guiTop + 16 + (int) sliderY, 0, 182, 12, 15);
 
         if (!clicked && Mouse.isButtonDown(0))
         {
@@ -429,7 +432,7 @@
                     item.dragging = true;
                 }
             }
-            if (adjustedMouseX <= 191 && adjustedMouseX >= 180)
+            if (adjustedMouseX <= 191 + guiOffset && adjustedMouseX >= 180 + guiOffset)
             {
                 if (enchantingPages != 0)
                 {
@@ -574,7 +577,7 @@
      */
     private GuiItem getItemFromPos(int x, int y)
     {
-        if (x < guiLeft + 35 || x > guiLeft + xSize - 32)
+        if (x < guiLeft + guiOffset + 35 || x > guiLeft + guiOffset + xSize - 32)
         {
             return null;
         }
@@ -604,7 +607,7 @@
 
         if (eventDWheel != 0)
         {
-            if (mouseX >= 35 && mouseX <= xSize - 32 || mouseX >= 180 && mouseX <= 192)
+            if (mouseX >= 35 + guiOffset && mouseX <= xSize + guiOffset - 32 || mouseX >= 180 + guiOffset && mouseX <= 192 + guiOffset)
             {
                 if (mouseY >= 15 && mouseY <= 87)
                 {
@@ -634,8 +637,8 @@
     public void initGui()
     {
         super.initGui();
-        buttonList.add(new GuiIcon(0, guiLeft + 9, guiTop + 40, "E").customTexture(0));
-        buttonList.add(new GuiIcon(1, guiLeft + 9, guiTop + 60, "R").customTexture(0));
+        buttonList.add(new GuiIcon(0, guiLeft + guiOffset + 9, guiTop + 40, "E").customTexture(0));
+        buttonList.add(new GuiIcon(1, guiLeft + guiOffset + 9, guiTop + 60, "R").customTexture(0));
         final String s = "Vanilla";
         buttonList.add(new GuiButton(2, guiLeft + xSize + 10, guiTop + 5, fontRenderer.getStringWidth(s) + 10, 20, s));
 
@@ -687,7 +690,7 @@
         {
             this.enchantments = enchantments;
 
-            enchantmentArray = convertMapToGuiItems(enchantments, 35 + guiLeft, 15 + guiTop);
+            enchantmentArray = convertMapToGuiItems(enchantments, 35 + guiOffset + guiLeft, 15 + guiTop);
 
             sliderIndex = enchantingPages = 0;
             clicked = sliding = false;
@@ -696,7 +699,7 @@
 
         if (dirty)
         {
-            final ArrayList<GuiItem> temp = convertMapToGuiItems(enchantments, 35 + guiLeft, 15 + guiTop);
+            final ArrayList<GuiItem> temp = convertMapToGuiItems(enchantments, 35 + guiOffset + guiLeft, 15 + guiTop);
 
             for (final GuiItem item : enchantmentArray)
             {
