--- ../eplus/gui/GuiModTable.java
+++ ../work/eplus/gui/GuiModTable.java
@@ -12,6 +12,7 @@
 import net.minecraft.enchantment.Enchantment;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.entity.player.InventoryPlayer;
+import net.minecraft.util.ResourceLocation;
 import net.minecraft.world.World;
 
 import org.lwjgl.input.Mouse;
@@ -89,7 +90,7 @@
                 GL11.glDisable(GL11.GL_LIGHTING);
                 GL11.glDisable(GL11.GL_DEPTH_TEST);
                 GL11.glColor4f(1.0f, 1.0f, 1.0f, 1.0f);
-                mc.renderEngine.bindTexture(texture);
+                mc.renderEngine.func_110577_a(texture);
                 drawTexturedModalRect(xPosition, yPosition, 8 + textureIndex * 16, 182, width, height);
             }
         }
@@ -278,7 +279,7 @@
 
     private boolean dirty = false;
 
-    private final String texture = "/assets/eplus/textures/gui/enchant.png";
+    private final ResourceLocation texture = new ResourceLocation("eplus", "textures/gui/enchant.png");
 
     private String error = "";
 
@@ -376,7 +377,7 @@
     protected void drawGuiContainerBackgroundLayer(float f, int i, int j)
     {
 
-        mc.renderEngine.bindTexture(texture);
+        mc.renderEngine.func_110577_a(texture);
         drawTexturedModalRect(guiLeft, guiTop, 0, 0, xSize, ySize);
 
         for (final GuiItem item : enchantmentArray)
@@ -402,7 +403,7 @@
         final int adjustedMouseX = par1 - guiLeft;
         final int adjustedMouseY = par2 - guiTop;
 
-        mc.renderEngine.bindTexture(texture);
+        mc.renderEngine.func_110577_a(texture);
         GL11.glEnable(GL11.GL_LIGHTING);
         GL11.glEnable(GL11.GL_DEPTH_TEST);
         GL11.glColor4f(1.0f, 1.0f, 1.0f, 1.0f);
