--- ../src/eplus/inventory/ContainerEnchantTable.java
+++ ../src_work6/eplus/inventory/ContainerEnchantTable.java
@@ -158,7 +158,7 @@
         {
             putStackInSlot(0, tileEnchantTable.itemInTable);
             PacketDispatcher.sendPacketToAllAround(tileEnchantTable.xCoord, tileEnchantTable.yCoord, tileEnchantTable.zCoord, 64, tileEnchantTable.worldObj.getWorldInfo()
-                    .getDimension(), tileEnchantTable.getDescriptionPacket());
+                    .getVanillaDimension(), tileEnchantTable.getDescriptionPacket());
             onCraftMatrixChanged(tableInventory);
         }
     }
@@ -342,9 +342,9 @@
     }
 
     @Override
-    public void onCraftGuiClosed(EntityPlayer par1EntityPlayer)
-    {
-        super.onCraftGuiClosed(par1EntityPlayer);
+    public void onContainerClosed(EntityPlayer par1EntityPlayer)
+    {
+        super.onContainerClosed(par1EntityPlayer);
 
         for (int i = 0; i < tableInventory.getSizeInventory(); i++)
         {
